server/services/participants-helper.js

```diff
@@ -574,7 +574,7 @@ class ParticipantsFinder {
     this.hiredGlobalJoin = 'hiredGlobalJoin';
     this.siteJoin = 'siteJoin';
     this.siteDistanceJoin = 'siteDistanceJoin';
-    this.rosStatuses = 'rosStatuses';
+    this.rosStatuses = user.isHA || user.isEmployer ? 'rosStatuses' : 'ros_infos';
```

Comment:
Can we rename this field on the back end prior to sending it to the front end? Getting this inconsistency fixed as close to the source as possible would be best I think

Reply:
We can rename rosStatuses to ros_infos but we cannot rename ros_infos to rosStatuses.

Reply:
The service method that queries the view returns `ros_infos` right? Can we get it to modify that key to `rosStatuses` before sending it to the FE?

Reply:
It does change it to rosStatuses before it sends it to the FE (happens at line 75)- the sorting comes before that.

Reply:
Oh jeez sorry I thought this change was on the front end 🥴 

A comment might be all we can do, but is there anything we can do to make it more clear that we're calling it `ros_infos` because of the constraint on the view rather than because it's MoH

Reply:
I think a comment is best. Technically there's also the table variable in the constructor for PartticipantsFinder (which gets changed to the view for other user) that's really the source of the difference, but that one gets modified in a few places (when attaching a join for non-MoH) so I'm less inclined to change it (rosStatuses stays constant).